### DISCUSSION TITLE
Nightly Release 1.2.4 Url Format Changed?

### DISCUSSION THREAD

**Discussion by GengGode** [rustdesk/rustdesk#7853] [Q&A]
### Bug Description

old host_name:port format in 1.2.1 is error
http://host_name:port don't connect!

### How to Reproduce

need a host_name, don`t ip

### Expected Behavior

url should can host_name:port

### Operating system(s) on local side and remote side

win11 -> win11

### RustDesk Version(s) on local side and remote side

1.2.3-2 -> 1.2.4

### Screenshots

this 1.2.4
![image](https://github.com/rustdesk/rustdesk/assets/47711102/0f0850c5-5860-4891-8293-75787046d973)
![image](https://github.com/rustdesk/rustdesk/assets/47711102/9b59ff6a-020a-42eb-9793-24ee4212412c)

this old 1.2.3-2
![image](https://github.com/rustdesk/rustdesk/assets/47711102/af344b86-efc0-44a0-ac35-c61a7ee5c336)
![image](https://github.com/rustdesk/rustdesk/assets/47711102/1f72ff04-c04f-45d1-ab0e-4949c8061aa9)

### Additional Context

I want to use the dual screen function, so I upgraded to 1.2.4 on the control end, but there was no dual screen display at the same time. So I suspected that the controlled end also needed to upgrade, but after upgrading, I found that I couldn't connect properly, including changing the domain name port to http://domain name port. I am still trying it out
Now I'm trying to return all of them

**Comment by fufesou** [comment]
Confirmed. I'll fix it.

**Reply by GengGode** [reply]
It seems like it was just a verification error. When upgrading from 1.2.3-2 to 1.2.4, everything was normal

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/7856
